### PR TITLE
Edit import command to support optional modname

### LINKED ISSUE #193: Fix import command to support optional modname

### PR DESCRIPTION
**Pull request by jtooya**
closes #193

### CODE REVIEW

File: importTestCases/README.txt [hunk 1]
```diff
@@ -8,8 +8,9 @@ Invalid Input: Attempts to replace invalid inputs for contact info*, skips the c
 Different order: Imports all contacts
 Unnecessary column: Imports all contacts, ignores extraneous columns
 
-*Contact info refer to either the email, telegram handle or phone number. If any of these fields are invalid or empty, the app will replace them with default values when importing
+*Contact info refer to either the email, telegram handle, phone number or module name. If any of these fields are invalid or empty, the app will replace them with default values when importing
 
 Default phone number: 0000 0000
 Default email: Default@email.net
```
Comment (SlothyCat) [inline importTestCases/README.txt:14]: should it be default@email.net as shown in your JsonAdaptedPerson.java?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -274,7 +274,7 @@ Format: `import`
 </box>
 
 * If the program comes across a contact that has an empty or invalid contact info, it will attempt to fill in the info
-* It will fill in the info with placeholder values (0000 0000, default@email.net, @DefaultHandle)
+* It will fill in the info with placeholder values (0000 0000, default@email.net, @DefaultHandle, AA0000)
```
Comment (SlothyCat) [inline docs/UserGuide.md:277]: The default telegram handle in your JsonAdaptedPerson.java seems to be @defaulthandle but over here the D and H are capitalised. Does the space between the 0s for phone number affect anything as the one in JsonAdaptedPerson.java does not seem to have any spaces?

**Review by SlothyCat — DISMISSED** [review]
LGTM other than some comments on the default values

File: docs/UserGuide.md (hunk 2 above)
Comment (jtooya) [inline docs/UserGuide.md]: space has no impact on anything other than identifying it as having 8 digits, as is the common format for many phone numbers

**Review by zienxu — APPROVED** [review]
lgtm!

**Pull request merged**